qubes_config/global_config/updates_handler.py

```diff
@@ -429,7 +429,8 @@ def _needs_updatevm_filter(vm):
             return False
         if getattr(vm, 'template', None):
             return False
-        return not vm.is_networked()
+        return bool(vm.features.check_with_template(
+            'service.updates-proxy-setup', False))
```

Comment:
As noted in the other PR - this service is implicite enabled in templates, so it wont work this way directly. But it should be okay if you change `False` to `vm.klass == TemplateVM`

Reply:
done

Reply:
you changed that in a wrong commit...